docusaurus.config.ts

```diff
@@ -28,6 +29,7 @@ const config: Config = {
           routeBasePath: '/',
         },
         docs: {
+          beforeDefaultRehypePlugins: [addTrailingSlash],
```

Comment:
Ik snap dat `config.url` niet echt werkt at this stage, maar je zou eventueel de site URL kunnen injecteren zodat die niet hardcoded in de plugin zit.

```suggestion
          beforeDefaultRehypePlugins: [addTrailingSlash(url, config.url)],
```